Lib/idlelib/NEWS.txt

```diff
@@ -2,6 +2,7 @@ What's New in IDLE 3.8.0 (since 3.7.0)
 Released on 2019-10-20?
 ======================================
 
+bpo-35598: Improve config_key with PEP8 names.
```

Comment:
I now remember why I made these additions separately.  This is correct, but merging it would conflict with a similar diff adding a different item to the top of the list.  So before merging I should temporarily copy this to my list of needed news items until the other is merged.

Reply:
That makes sense.  I'll leave adding the News item until a PR is ready to merge.